chart/values.yaml

```
@@ -19,6 +19,8 @@ appConfigValues:
   googleCloudStorageProjectId: ""
   ## @param appConfigValues.awsS3Region If the bucket is S3, the region to be used
   awsS3Region: ""
+  ## @param appConfigValues.bigqueryOauth2ClientId If the bucket is S3, the region to be used
```

Please, update the docu of the variable accordingly....
```suggestion
  ## @param appConfigValues.bigqueryOauth2ClientId bla bla bla 
```